node-check/osgvo-advertise-base

```diff
@@ -31,7 +31,7 @@
 # OSG_GLIDEIN_VERSION is an ever-increasing version of the glideins.
 # This can be used by negotiators or users, for example to match
 # against a glidein newer than some base with new features.
-OSG_GLIDEIN_VERSION=622
+OSG_GLIDEIN_VERSION=624
```

Comment:
Only even version numbers?

Reply:
I'm skipping 623 because I'm using 623 in a different PR